homeassistant/base/Dockerfile

```diff
@@ -112,3 +112,27 @@ RUN apk add --no-cache \
     && pip3 install --no-cache-dir https://github.com/wayfair/pymssql/archive/v2.1.3.0.0.1.zip \
     && pip3 install --no-cache-dir cchardet uvloop aiodns \
     && apk del .build-dependencies
+    
+####
+## Temporary Fix for Mi Flora. Remove on release of Alpine 3.7
+RUN apk add --no-cache \
+        gcc g++ make musl-dev consolekit dbus dbus-dev libusb-compat-dev eudev-dev \
```

Comment:
Amazing. Do we need dbus/consolekit here?

Reply:
yep, both are dependencies for bluez